wallet-new/src/Cardano/Wallet/Kernel/DB/Util/IxSet.hs

```diff
@@ -154,6 +156,10 @@ size = IxSet.size . unwrapIxSet
 fromList :: Indexable a => [a] -> IxSet a
 fromList = WrapIxSet . IxSet.fromList . coerce
 
+-- | Destructs an 'IxSet' into a list
+toList :: IxSet a -> [a]
+toList (WrapIxSet wrapped) = map coerce (IxSet.toList wrapped)
```

Comment:
We don't need this anymore as this was added only for debugging some tests.

Reply:
So then why did you leave it? :D